### PR TITLE
Turned on partial_success by default and removed obsolete comment of support availability.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

File: lib/fluent/plugin/out_google_cloud.rb [hunk 1]
```diff
@@ -307,8 +307,7 @@ module InternalConstants
 
     # Whether valid entries should be written even if some other entries fail
     # due to INVALID_ARGUMENT or PERMISSION_DENIED errors when communicating to
-    # the Stackdriver Logging API. This is highly recommended. Right now this
-    # only works with the REST path (use_grpc = false).
+    # the Stackdriver Logging API. This is highly recommended.
     config_param :partial_success, :bool, :default => false
```
Comment (igorpeshansky) [inline lib/fluent/plugin/out_google_cloud.rb]: Would it make sense to also flip the default to `true`? This shouldn't break existing customers, right?

Comment (qingling128) [reply lib/fluent/plugin/out_google_cloud.rb]: Good idea. Changed.

**Comment by qingling128** [comment]
Adjusted unit tests. PTAL.

**Comment by qingling128** [comment]
Created b/72756346 to enhance the unit tests.

**Pull request merged**